spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/annotation/LoadBalancerClientConfiguration.java

```diff
@@ -143,6 +143,15 @@ public ServiceInstanceListSupplier weightedServiceInstanceListSupplier(Configura
 					.build(context);
 		}
 
+		@Bean
```

Comment:
Thanks. We should also have one with the non-reactive discovery client (`@ConditionalOnBean(DiscoveryClient.class`). Also, please provide documentation for your feature.